README.md

```diff
@@ -186,12 +186,12 @@ Console.WriteLine(deathRecord.ToJSON());
 ```
 
 #### Return Coding
-An example of producing a `CodingResponseMessage` for handling the returned message from NCHS containing coded causes. For a complete example, [click here](https://github.com/nightingaleproject/vrdr-dotnet/blob/master/doc/Messaging.md#return-coding).
+An example of producing a `CauseOfDeathCodingResponseMessage` for handling the returned message from NCHS containing coded causes. For a complete example, [click here](https://github.com/nightingaleproject/vrdr-dotnet/blob/master/doc/Messaging.md#return-coding).
```

Comment:
This example points to a CodingResponseMessage. Not sure if the IG changes are available or pushed to a master branch, so syncing this may have to wait until Saul publishes it.